### PR TITLE
Added support for VLAN enabled networks

### PR DESCRIPTION
**Pull request by gshimansky**

### CODE REVIEW

**Comment by ifilippov** [comment]
Looks good to me, however: maybe divide for two commits? For packet.go and for NAT separately?
Commit name is bad, because we have already support VLANs, more details are required.

**Comment by ifilippov** [comment]
And comments! First word in comment must be the same as function name!!!

File: examples/nat/main/Makefile [hunk 1]
```diff
@@ -6,6 +6,6 @@ PATH_TO_MK = ../../../mk
 IMAGENAME = yanff-nat
 EXECUTABLES = nat
 
-nat: ../config.go ../translation.go ../portalloc.go
+nat: ../config.go ../translation.go ../portalloc.go ../cksum.go ../util.go
```
Comment (mbulatova) [inline examples/nat/main/Makefile]: No file with name util.go added in this commit

Comment (gshimansky) [reply examples/nat/main/Makefile]: Sorry, forgot to add it. It is fixed in pull request #111

**Pull request merged**